lib/app/components/atoms/chat_bubble.dart

```diff
@@ -14,12 +16,15 @@ class ChatBubble extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BubbleSpecialThree(
-      text: text,
-      isSender: isSender,
-      color: colorChatBubbleBg, //TODO: 将来的にdark mode対応予定
-      tail: true,
-      textStyle: const TextStyle(color: colorChatBubbleText, fontSize: 16),
+    return cb.ChatBubble(
+      clipper: isSender
+          ? ChatBubbleClipper4(type: BubbleType.sendBubble)
+          : ChatBubbleClipper4(type: BubbleType.receiverBubble),
```

Comment:
```suggestion
          ? ChatBubbleClipper1(type: BubbleType.sendBubble)
          : ChatBubbleClipper1(type: BubbleType.receiverBubble),
```
`ChatBubbleClipper4`だと吹き出しの三角(◀️)の開始位置が微妙だったので、`ChatBubbleClipper1`にしてみましょう。

Reply:
ご指摘の通り修正しました！